src/NuGet.Core/NuGet.Build.Tasks.Pack/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,2 @@
+NuGet.Build.Tasks.Pack.IPackTaskLogic.GetPackArgs(NuGet.Build.Tasks.Pack.IPackTaskRequest<NuGet.Commands.IMSBuildItem> request, NuGet.Commands.PackCommand.WarningPropertiesCollection warningPropertiesCollection) -> NuGet.Commands.PackArgs
```

Comment:
I think this is something that should have probably been removed.

UsePublicApiAnalyzer is false for this project. 

Probably got commited mistakenly by someone.

Reply:
Should I update `tools-local\ship-public-apis` tool to prevent this in the future?

Reply:
Removed for now.

Reply:
I don't think we ned to update the tooling. 
This file was committed by mistake, and the tooling works well enough.